ui/jasmine_test/spec/cookie-service.spec.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CookieService} from '../../src/utils/CookieService';
+
+describe('Testing that the Cookie Service getCookieValue()', () => {
+    
+    const cookieService = new CookieService(); 
```

Comment:
You could combine the `describe` into one, create `cookieService` and call `makeUserInfoCookie` in a `beforeEach`. This setup is common between both test blocks

Reply:
moved all to one `describe` -- the set up is only required once so I chose not to use the `beforeEach`